Gemfile.lock

```diff
@@ -123,6 +125,7 @@ GEM
     yard (0.9.34)
 
 PLATFORMS
+  aarch64-linux
```

Comment:
Are these supposed to be added to the gemfile?

Reply:
No, I've reverted that change. Thanks for catching that!